src/foam/u2/wizard/data/PrerequisiteLoader.js

```diff
@@ -102,7 +102,7 @@ foam.CLASS({
             `prerequisiteCapabilityId: ${this.prerequisiteCapabilityId}'s data returns null for the path ${this.loadFromPath.toString()}`,
           );
           if ( this.of ) {
-            return this.of.create({}, this);
+            return initialData || this.of.create({}, this);
```

Comment:
This is definitely better, but if you're making use of this that means this prerequisite sometimes has data to update a property on this wizardlet's `data` and sometimes doesn't. Is this what you expect?

Reply:
yes its expected that the property on the prereq is not always available, in this case the username on signin which is required only when the email is associated with multiple users